tools/pubsys/src/repo.rs

```diff
@@ -753,6 +763,13 @@ mod error {
             source: tough::error::Error,
         },
 
+        /// The library failed to instantiate 'tokio Runtime'.
+        #[snafu(display("Unable to create tokio runtime: {}", source))]
+        RuntimeCreation {
+            source: std::io::Error,
+            backtrace: Backtrace,
```

Comment:
Why do we need a backtrace here? I also think you need to enable the `backtraces` feature flag for the snafu crate for this to have any effect? https://docs.rs/snafu/0.6.3/snafu/guide/feature_flags/index.html#backtraces

Reply:
I only had it there to stay consistent with our implementation in [`tough-kms`](https://github.com/awslabs/tough/blob/develop/tough-kms/src/error.rs#L19), but removing it doesn't cause any issues that I'm aware of.